agents/insurance-solution/README.md

```diff
@@ -0,0 +1,74 @@
+# Task Automation Using Agents and Knowledge Base for Amazon Bedrock - Insurance Agent. 
+---
+
+## Content
+- [Overview](#overview)
+- [Agents and Knowledge Base Architecture](#agents-and-knowledge-base-architecture)
+- [Deployment Guide](#deployment-guide)
+- [Testing and Validation](#testing-and-validation)
+- [Clean Up](#clean-up)
+
+## Overview
+
+You can now use [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html) and [Knowledge base for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/knowledge-base.html) to configure specialized agents that seamlessly execute actions based on user input and your organization's data. These managed agents play conductor, orchestrating interactions between foundation models, API integrations, user conversations, and knowledge bases loaded with your data. Agents and knowledge bases allow you to **build on existing enterprise resources** to enhance user experience and automate repetitive tasks.
+
+This insurance agent sample solution combines an Agent and Knowledge base for Amazon Bedrock to automate insurance claim lifecycle actions like creating new claims, sending pending document reminders for open claims, gathering claims evidence, and searching for information on existing claims.
+
+### Demo Recording
+[<img src="design/thumbnail.png" width="100%">](https://www.youtube.com/watch?v=bv4XV7epeik "Task Automation Using Agents for Amazon Bedrock - YouTube")
+
+## Agents and Knowledge Base for Amazon Bedrock
+
+### Agents and Knowledge Base Functionality
+Agents and Knowledge Base for Amazon Bedrock work together to provide the following set of capabilities:
+
+- **Task Decomposition:** Agents expand foundation models to comprehend user inquiries and dissect tasks into manageable steps for execution.
+- **Interactive Data Collection:** Agents engage in natural conversations to gather supplementary information from users.
+- **Task Execution:** Agents fulfill customer requests through series of reasoning steps and corresponding actions based on [ReAct prompting](https://www.promptingguide.ai/techniques/react).
+- **System Integration:** Agents make API calls to internal company systems to execute specific action groups.
+- **Data Querying:** Knowledge bases enhance accuracy and performance through fully-managed [retrieval augmented generation (RAG)](https://docs.aws.amazon.com/sagemaker/latest/dg/jumpstart-foundation-models-customize-rag.html) using customer specific data sources.
+- **Source Attribution:** Agents conduct source attribution, identifying and tracing the origin of information or actions.
+
+### Agents and Knowledge Base Architecture
+
+<p align="center">
+  <img src="design/agent-overview.png">
+  <em>Diagram 1: Agents and Knowledge Base for Amazon Bedrock Architecture Overview</em>
```

Comment:
the video is not connecting with the architecture highlighted here, which does focus on Bedrock native solutions. Are you considering the same agent?